### ISSUE TITLE
cannot install on macos

### ISSUE THREAD

**Issue by titouancreach** [memphisdev/memphis.js#197]
when running on mac, got the stack trace:

```
 ⨯ Error: Cannot find module '/Users/redacted/Documents/Code/Poc/redacted/node_modules/murmurhash3/lib/../build/default/murmurhash3'
Require stack:
- /Users/redacted/Documents/Code/Poc/redacted/node_modules/murmurhash3/lib/murmurhash3.js
- /Users/redacted/Documents/Code/Poc/redacted/node_modules/memphis-dev/lib/memphis.js 
```

g++ is installed:

```
$ /usr/bin/g++           
clang: error: no input files
```

other: 
  node 20.3.0

tried with bun and npm

**Comment by yanivbh1** [comment]
Hey @titouancreach ,
Thank you for opening an issue, we are checking it right now.

**Comment by daniel-davidd** [comment]
Hi, can you please share a bit about how you installed the Memphis SDK and what the code does?
I didn't manage to reproduce the problem on my mac.

**Comment by titouancreach** [comment]
Ok I found the problem,

I first started to install the sdk with bun (https://bun.sh/) and I got the message above. So then I installed with npm: "npm i" because I had the feeling that bun was the problem, but I didn't remove node_modules. 

This morning, I did, and installed everything with npm and it started working.
So the problem is that it doesn't work with bun, so nothing to do with you sdk.

Thanks <3
